lib/pack/aws/cache.ts

```diff
@@ -13,16 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-import { doWithRetry } from "@atomist/automation-client/lib/util/retry";
 import * as AWS from "aws-sdk";
 import * as fs from "fs-extra";
-import { GoalInvocation } from "../../api/goal/GoalInvocation";
-import { CacheConfiguration } from "../../api/machine/SoftwareDeliveryMachineOptions";
-import { GoalCacheArchiveStore } from "../../core/goal/cache/CompressingGoalCache";
+import {GoalInvocation} from "../../api/goal/GoalInvocation";
+import {CacheConfiguration} from "../../api/machine/SoftwareDeliveryMachineOptions";
+import {doWithRetry} from "../../client";
+import {GoalCacheArchiveStore} from "../../core";
```

Comment:
Avoid importing from index files in the same project. It has a tendency to cause mocha to succeed without running any tests. Just keep the original imports.